packages/rs-dpp/src/state_transition/mod.rs

```diff
@@ -1167,32 +1169,22 @@ impl StateTransitionStructureValidation for StateTransition {
         platform_version: &PlatformVersion,
     ) -> crate::validation::SimpleConsensusValidationResult {
```

Comment:
Maybe 

```suggestion
    ) ->  ConsensusValidationResult<bool> {
```

where bool:
* true - correct
* false - undecided

Reply:
This is outside the normal pattern and confusing. The validation result would pass, but the bool would say it's valid? It's three tiers and hurts my brain.